semantic_router/splitters/running_avg_sim.py

```diff
@@ -0,0 +1,55 @@
+from typing import List
+from semantic_router.splitters.base import BaseSplitter
+from semantic_router.encoders import BaseEncoder
+import numpy as np
+from semantic_router.schema import DocumentSplit
+
+class RunningAvgSimSplitter(BaseSplitter):
```

Comment:
rolling avg or running avg?

Reply:
I've removed this code for now. it actually works well, but it has an issue where it automatically assigns the first two documents to the same topic. Because of the way it works (by looking at differences between rolling average similarity scores and the next similarity score, we can't actually assign a score to the first two messages - We have the similarity score between those messages, but no other similarity score to compare to. 

The upshot is that it doesn't work nicely with the Conversations, particularly when we try and split a newer set of documents. The first new document is always set to have the same topic as the last document from the last time splitting was carried out. 

So I've removed it for now. I might look into a fix if we think it's necessary.

Reply:
But i've made the changes in my copies of the code! Now says Rolling.